themes/dkn/light/component.json

```diff
@@ -223,5 +223,31 @@
         }
       }
     }
+  },
+  "card": {
+    "color": {
+      "background": {
+        "value": "{container.color.background}",
+        "type": "color"
+      }
+    },
+    "border": {
+      "hover": {
+        "value": {
+          "color": "#BFBFBF",
+          "width": "2",
+          "style": "solid"
+        },
+        "type": "border"
+      },
```

Comment:
@griomike We have already included card.border.alert in themes/component.json. Therefore, it is not needed in themes/dkn/light/component.json.

Reply:
Shouldn't it be placed in **themes/dkn/light/** so that a dark mode version can also be created?

Reply:
@griomike 
Since “color.feedback.negative” is present in “themes/system.json,” this “color feedback token” uses the same color for both dark mode and light mode. Therefore, “card.border.alert” should not be included in “themes/dkn/light/component.json.” Is this reasonable?

Reply:
If the color gets inherited from the parent theme, and this behavior is kept if we ever decide to change the values for either light/dark mode, then it makes sense if we don't include this.

Reply:
@hiro-daikin Keitaro and I discussed the "card border alert" tokens. We think notification colors should exist only at the level of component light & dark modes — since these colors very often have to be adjusted to work in dark mode (although not always). 

I've provided examples from HERO Cloud in Figma.